src/pages/create.tsx

```diff
@@ -5,7 +5,7 @@ import withAuth from '@components/auth/withAuth';
 
 const Create = () => (
   <>
-    <SEO pageTitle="Create New" />
+    <SEO pageTitle="Create an NFT" />
     <Breadcrumb pageTitle="Create new NFT" />
```

Comment:
Should it be the same message? Create an NFT?

Reply:
It's not required. You can set a message on the SEO (window title) and other on the Breadcrumb